amazon_sqs_base_block.py

```diff
@@ -1,11 +1,13 @@
 import boto3
 from nio.block.base import Block
 from nio.properties import (VersionProperty, PropertyHolder, StringProperty,
-                            ObjectProperty, FileProperty)
+                            ObjectProperty)
 from nio.util.discovery import not_discoverable
 
 
 class AWSCreds(PropertyHolder):
+    region_name = StringProperty(
```

Comment:
Unless this is only available in one region, maybe a SelectProperty of strings so they can choose which region from a dropdown?